website/screens/components/grid/code/GridCodePage.tsx

```diff
@@ -0,0 +1,327 @@
+        content: (
+          <DxcTable>
+            <tr>
+              <th>Name</th>
+              <th>Default</th>
+              <HeaderDescriptionCell>Description</HeaderDescriptionCell>
+            </tr>
+            <tr>
+              <td>areaName: string</td>
+              <td></td>
+              <td>
+                Sets the name of an item so that it can be referenced by a
+                template created with the <Code>grid-template-areas</Code>{" "}
+                property.
+              </td>
+            </tr>
+            <tr>
+              <td>column: number | string | GridCell</td>
+              <td></td>
+              <td>
+                Sets the <Code>grid-column</Code> CSS property. See{" "}
+                <DxcLink
+                  newWindow
+                  href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-column"
+                >
+                  MDN
+                </DxcLink>{" "}
+                for further information.
+              </td>
+            </tr>
+            <tr>
+              <td>row: number | string | GridCell</td>
+              <td></td>
+              <td>
+                Sets the <Code>grid-row</Code> CSS property. See{" "}
+                <DxcLink
+                  newWindow
+                  href="https://developer.mozilla.org/en-US/docs/Web/CSS/grid-row"
+                >
+                  MDN
+                </DxcLink>{" "}
+                for further information.
+              </td>
+            </tr>
+            <tr>
+              <td>
+                placeSelf: 'auto' | 'start' | 'end' | 'center' | 'stretch' |
+                'baseline' | object
+              </td>
+              <td>
+                <Code>'auto'</Code>
+              </td>
+              <td>
+                Sets the <Code>place-self</Code> CSS property. See{" "}
+                <DxcLink
+                  newWindow
+                  href="https://developer.mozilla.org/en-US/docs/Web/CSS/place-self"
+                >
+                  MDN
+                </DxcLink>{" "}
+                for further information.
+              </td>
+            </tr>
+            <tr>
+              <td>as: keyof HTMLElementTagNameMap</td>
+              <td>
+                <Code>'div'</Code>
+              </td>
+              <td>Sets the a custom HTML tag.</td>
+            </tr>
+            <tr>
+              <td>children: ReactNode</td>
+              <td></td>
+              <td>Custom content inside the grid container.</td>
+            </tr>
+          </DxcTable>
+        ),
+      },
+    ],
+  },
+  {
+    title: "Examples",
+    subSections: [
+      {
+        title: "Basic usage",
+        content: <Example example={basic} defaultIsVisible />,
+      },
+      {
+        title: "Building an application layout",
+        content: <Example example={layout} defaultIsVisible />,
+      },
+    ],
+  },
+];
+
+const FlexCodePage = () => {
+  return (
```

Comment:
Please change here `FlexCodePage` to `GridCodePage`.

Reply:
Done!